src/main/java/de/tum/in/www1/artemis/domain/tutorialgroups/TutorialGroupSchedule.java

```diff
@@ -112,6 +112,12 @@ public boolean sameSchedule(TutorialGroupSchedule other) {
                 && Objects.equals(this.validToInclusive, other.validToInclusive) && Objects.equals(this.location, other.location);
     }
 
+    public boolean onlyLocationChanged(TutorialGroupSchedule other) {
```

Comment:
Adding new property to the class would require the developer to add the property here as well. Could we add that information somewhere so that no one would miss it?

Reply:
Added note for developers in 50b3fb850ad70daf7b23e31c83893ddcb334c79a